src/api/data.js

```diff
@@ -44,18 +44,16 @@ function userDetailsChangedChannel() {
 }
 
 function transformBooks(bookData) {
-  if (bookData.isbns.length === 0) {
-    bookData.isbns = [{}];
-  }
-  return { author: bookData.author, title: bookData.title, ISBN: bookData.isbns[0].isbn13 };
+  return { author: bookData.author, title: bookData.title, ISBN: bookData.primary_isbn13, coverImagePath: bookData.book_image };
```

Comment:
Try and run the linter once more, you'll get an error to reduce the characters in this line

`npm run lint`